runtime/src/tiered_storage/footer.rs

```diff
@@ -1,6 +1,6 @@
 use {
     crate::tiered_storage::{
-        error::TieredStorageError, file::TieredStorageFile, index::AccountIndexFormat,
+        error::TieredStorageError, file::TieredStorageFile, index::AccountIndex,
```

Comment:
The PR also piggy-back the rename of AccountIndexFormat to AccountIndex.

I can also have it in a separate PR if that makes it cleaner.

Reply:
Reverted this part.  It might make more sense to have it in a separate PR that changes all the Format enum names if needed.